### ISSUE TITLE
Refreshed authtoken cookie does not include Expires date

### ISSUE THREAD

**Issue by warpech** [StarcounterPrefabs/SignIn#44]
### Steps to reproduce
1. Start Launcher and SignIn
2. Authenticate as "admin" user (check "Remeber me")
3. Check that you receive an authtoken with "Expires" date
4. Reload the page
5. Check that HTML response contains regenerated authtoken without the "Expires" date
### Expected result

The regenerated authtoken cookie should include relevant "Expires" date.
### Actual result

The regenerated authtoken cookie does not include the "Expires" date. This causes a problem, because from now on this session is not persistent. Browser cookie without "Expires" date as temporary (aka "browser session") and will delete it when the browser is closed.

I discovered this while working on #40.
##### Involved apps' version

StarcounterPrefabs/Launcher@5.0.0-develop.1
SignIn https://github.com/StarcounterPrefabs/SignIn/commit/ceb08ff34d6856a078bb348b7a0e3e3112f450b2
##### Browsers affected

- [x] Chrome
- [x] Firefox
- [ ] Edge (did not test)
- [ ] Safari 9 (did not test)
- [ ] Safari 8 (did not test)
- [ ] IE 11 (did not test)
### Screenshot

![screen](https://cloud.githubusercontent.com/assets/566463/18706994/67f9359e-7ff4-11e6-9114-1eb59b1b2ad8.gif)

**Comment by warpech** [comment]
@diliger pls take this when you have time

**Comment by diliger** [comment]
Well, after some research i figured out such behavior:

1. SignIn with RememberMe, receive Cookie in response with expiration time
![image](https://cloud.githubusercontent.com/assets/15857369/20582966/f076907c-b1f6-11e6-84af-c445dc7ceea6.png)

2. But in document.cookie variable I see only cookie name and value, without any information about expiration or other parameters
![image](https://cloud.githubusercontent.com/assets/15857369/20582975/0d178ef2-b1f7-11e6-89b6-4f3df6b3e6b1.png)

3. For any next request browser sends only cookies name and value without parameters
![image](https://cloud.githubusercontent.com/assets/15857369/20582984/29e0bb1c-b1f7-11e6-9206-3e2f4fbcf716.png)

4. On server side auth cookie got refreshed and that removes **expires** parameter
![image](https://cloud.githubusercontent.com/assets/15857369/20582992/3df40262-b1f7-11e6-86f5-8827070d8496.png)

Actually the problem is noted as TODO in comments:
```
//TODO: refreshing cookie removes the Expires date
//store the Expires date in Simplified and use it here
Handle.AddOutgoingCookie(cookie.Name, cookie.GetFullValueString());
```

**Comment by diliger** [comment]
So, what may be the way to fix it?

**Comment by warpech** [comment]
In `SetAuthCookie` there is a `cookie.Expires` call that sets the expiry date.

I guess it is enough to add `cookie.Expires` in `RefreshAuthCookie`. You will need to store the token expiry date in the `Expires` property of https://github.com/StarcounterPrefabs/Simplified/blob/master/Ring5/User/SystemUserTokenKey.cs - this is the purpose why that property was added.

**Comment by warpech** [comment]
This will only work with Starcounter nightly, SignIn develop - but that's fine.

**Comment by diliger** [comment]
please review my pull request #55

**Comment by warpech** [comment]
It seems to me that there are still two problems: 

1. When I sign in and reload the page, a new token string replaces the previous token. I think that this might lead to race conditions - if I open my app in 5 browser tabs simultaneusly, they might affect each other tokens.
2. The regenerated token does not extend Expires date. I think this is wrong. The regenerated token should have fresh "expires" date (7 or 30 days from now).

Whan I am saying is basically is that a `SystemUserTokenKey` class should be considered readonly for all the columns other than `LastUsed`. Refreshing the token means generating a new token.

@diliger this is just my opinion. Do you agree?

**Comment by diliger** [comment]
Can it be closed now?

**Comment by warpech** [comment]
Fixed in PR #55
